Resources/config/yui.yml

```diff
@@ -132,6 +132,7 @@ system:
                         - 'ez-templatebasedview'
                         - 'ez-actionbarview'
                         - 'ez-locationviewviewtabview'
+                        - 'ez-locationviewviewdetailstabview'
```

Comment:
to be consistent with `ez-locationviewviewtabview`, the detail tab view should be called `eZ.LocationViewDetailsTabView` and should be defined in the `ez-locationviewdetailstabview` module.